python/rainbow/simulators/prox_soft_bodies/types.py

```diff
@@ -252,6 +252,10 @@ def __init__(self):
             0.1  # Any geometry within this distance generates a contact point.
         )
         self.resolution = 64  # The number of grid cells along each axis in the signed distance fields.
+        self.proximal_solver = {
```

Comment:
It might be a little challenging to have this "matrix" of parallel/non-parallel vs gauss-seidel/Jacobi etc.

For instance, I get the impression that hybrid may not depend on parallel at all?

Reply:
Another way could be to simply have "names" for each solver type that is supported

gauss-seidel, parallel-gauss-seidel, jacobi, parallel-jacobi, parallel-jacobi-hybrid and so on?